backport/event.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/sethvargo/go-githubactions"
+)
+
+func UnmarshalEventData(ctx *githubactions.GitHubContext, v any) error {
+	if ctx.EventPath == "" {
+		return fmt.Errorf("event path is empty")
+	}
+
+	eventData, err := os.ReadFile(ctx.EventPath)
```

Comment:
Nit: I think there's a more modern way to handle this check -> https://pkg.go.dev/os#IsNotExist
```suggestion
eventData, err := os.ReadFile(ctx.EventPath)
if err != nil && !errors.Is(err, fs.ErrNotExist) {
    return fmt.Errorf("could not read event file: %w", err)
}
```

Reply:
true